projects/js-packages/publicize-components/src/components/share-limits-bar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { RecordMeterBar, Text } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+import styles from './styles.module.scss';
+
+export type ShareLimitsBarProps = {
+	currentCount: number;
+	scheduledCount: number;
+	activeConnectionsCount?: number;
+	maxCount: number;
+	text?: string;
+	className?: string;
+};
+
+export const ShareLimitsBar: React.FC< ShareLimitsBarProps > = ( {
+	currentCount,
+	maxCount,
+	scheduledCount,
+	activeConnectionsCount,
+	text,
+	className,
+} ) => {
```

Comment:
In fact, could we do:

```suggestion
export const ShareLimitsBar = ( {
	currentCount,
	maxCount,
	scheduledCount,
	activeConnectionsCount,
	text,
	className,
} : ShareLimitsBarProps ) => {
```
And have the return type inferred?

These are probably stylistic choices, which is why we could do with some lint rules.

Reply:
Done. Thanks. Yeah, lint rule should help.